peer/core.yaml

```diff
@@ -335,6 +335,13 @@ ledger:
       # Deploy chaincodes into the genesis block
       chaincodes:
 
+        noop_syscc:
+          path: github.com/hyperledger/fabric/core/system_chaincode/noop
+          type: GOLANG
+          constructor:
+            args:
+
+
```

Comment:
@tamasblummer understood the data is available within the node environment and thus to many entities with access to the peer itself.  My concern is this seems to expose that data to remote clients as well without any obvious ACL considerations.  But since we are in agreement this is an optional and off by default component, I don't see the harm.